FastMember/MemberSet.cs

```diff
@@ -65,6 +65,25 @@ internal Member(MemberInfo member)
             this.member = member;
         }
         /// <summary>
+        /// The ordinal of this member among other members.
+        /// Returns -1 in case the ordinal is not set.
+        /// </summary>
+        public int Ordinal
+        {
+            get
+            {
+                var ordinalAttr = member.CustomAttributes.FirstOrDefault(p => p.AttributeType == typeof(OrdinalAttribute));
+
+                if (ordinalAttr == null)
+                {
+                    return -1;
+                }
+
+                // OrdinalAttribute class must have only one constructor with a single argument.
+                return Convert.ToInt32(ordinalAttr.ConstructorArguments.First().Value);

```

Comment:
According to the comment, this should be a Single() call instead of a First() call.

Reply:
Fixed! Thanks for catching.